src/main/java/com/zaxxer/hikari/HikariConfig.java

```diff
@@ -665,6 +665,7 @@ public void setMetricRegistry(Object metricRegistry)
          metricRegistry = getObjectOrPerformJndiLookup(metricRegistry);
 
          if (!safeIsAssignableFrom(metricRegistry, "com.codahale.metrics.MetricRegistry")
+             && !(safeIsAssignableFrom(metricRegistry, "io.dropwizard.metrics5.MetricRegistry"))
              && !(safeIsAssignableFrom(metricRegistry, "io.micrometer.core.instrument.MeterRegistry"))) {
             throw new IllegalArgumentException("Class must be instance of com.codahale.metrics.MetricRegistry or io.micrometer.core.instrument.MeterRegistry");
```

Comment:
This exception message is now out of date

Reply:
Nice, thanks.  Fixed and amended the commit with ec214a23.